docs/coreutils-experiments.md

```diff
@@ -113,3 +122,7 @@ This document is structured as a series of FAQs:
     * **od:** `--sym-args 0 3 10 --sym-files 2 12 --sym-stdin 12 --sym-stdout`
     * **pathchk:** `--sym-args 0 1 2 --sym-args 0 1 300 --sym-files 1 8 --sym-stdin 8 --sym-stdout`
     * **printf:** `--sym-args 0 3 10 --sym-files 2 12 --sym-stdin 12 --sym-stdout`
+
+1. How did you measure coverage?
+
+   As mentioned in the paper, we replayed the test inputs generated by KLEE and used `gcov` to measure coverage.  One thing to take into account is that `gcov` does not record coverage on `_exit`.
```

Comment:
Good to know! So as `_exit` skips all on_exit handlers,  `__gcov_flush()` is not called and no coverage output is written.
This is bad for testcases exercising failure handling.

Maybe you could extend the last sentence a bit.